From 85d452490714b8de41248e40e28b3da2e0ce6c04 Mon Sep 17 00:00:00 2001
From: Pranav Vashi <neobuddy89@gmail.com>
Date: Tue, 11 Aug 2026 09:51:42 +0000
Subject: [PATCH] SystemUI: Fix QS tile dialog background color

[Port to A16] Theme.SystemUI.Dialog colorBackground uses @androidprv:color/ reference syntax in A16; same value swap to notification_divider_color.

Change-Id: I4c07a10004daf7e56c88e13162e9f23de858ada1
---
 packages/SystemUI/res/values/styles.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/res/values/styles.xml b/packages/SystemUI/res/values/styles.xml
index e3adee94733b..2d13a23d0b42 100644
--- a/packages/SystemUI/res/values/styles.xml
+++ b/packages/SystemUI/res/values/styles.xml
@@ -611,7 +611,7 @@
         <item name="android:buttonBarPositiveButtonStyle">@style/Widget.Dialog.Button</item>
         <item name="android:buttonBarNegativeButtonStyle">@style/Widget.Dialog.Button.BorderButton</item>
         <item name="android:buttonBarNeutralButtonStyle">@style/Widget.Dialog.Button.BorderButton</item>
-        <item name="android:colorBackground">@androidprv:color/materialColorSurfaceBright</item>
+        <item name="android:colorBackground">@color/notification_divider_color</item>
         <item name="android:alertDialogStyle">@style/ScrollableAlertDialogStyle</item>
         <item name="android:buttonBarStyle">@style/ButtonBarStyle</item>
         <item name="android:buttonBarButtonStyle">@style/Widget.Dialog.Button.Large</item>
-- 
2.43.0

